src/tests/ascent/t_ascent_rover.cpp

```diff
@@ -19,70 +19,307 @@
+    extracts["e1/params/absorption"] = "radial";
+    extracts["e1/params/emission"] = "radial";
+    extracts["e1/params/filename"] = query_output_file;
+    extracts["e1/params/blueprint"] = "json";
+
+    conduit::Node actions;
+    // add the pipeline
+    conduit::Node &add_extracts = actions.append();
+    add_extracts["action"] = "add_extracts";
+    add_extracts["extracts"] = extracts;
+
+    //
+    // Run Ascent
+    //
+
+    Ascent ascent;
+
+    Node ascent_opts;
+    ascent.open(ascent_opts);
+    ascent.publish(data);
+    ascent.execute(actions);
+    ascent.close();
+
+    const std::string full_outfile_name = query_output_file + "100.cycle_000100.root";
+
+    Node load_mesh, verify_info;
+    conduit::relay::io::blueprint::load_mesh(full_outfile_name, load_mesh);
+    EXPECT_TRUE(conduit::blueprint::mesh::verify(load_mesh, verify_info));
+
+    const std::string image_output_path = prepare_output_dir();
+    const std::string image_output_base =
+        conduit::utils::join_file_path(image_output_path, "tout_rover_xray_blueprint");
+
+    const std::string image_output_image_prefix = image_output_base + "{cycle:d}";
+
+    render_blueprint_result("intensities", image_output_image_prefix, load_mesh);
+
+    // TODO we need the render to make an interesting picture. This will be accomplished
+    // by working on the basic mesh output and changing the order of the dimensions.
+    // TODO we will need to change the baseline when we are making good renders.
+    EXPECT_TRUE(check_test_image(image_output_base, 0.01f, "100"));
+
+    std::string msg = "TODO we need a good description here";
+    ASCENT_ACTIONS_DUMP(actions, image_output_base, msg);
+}
+
+//-----------------------------------------------------------------------------
+TEST(ascent_rover, test_xray_blueprint_curv3d)
+{
+    // the vtkm runtime is currently our only rendering runtime
+    Node n;
+    ascent::about(n);
+    // only run this test if ascent was built with vtkm support
+    if(is_vtk_disabled(n))
+    {
+        return;
+    }
+
+    //
+    // Open an example mesh.
+    //
     Node data, verify_info;
-    conduit::blueprint::mesh::examples::braid("hexs",
-                                              EXAMPLE_MESH_SIDE_DIM,
-                                              EXAMPLE_MESH_SIDE_DIM,
-                                              EXAMPLE_MESH_SIDE_DIM,
-                                              data);
+    const std::string root_file = 
+        conduit::utils::join_file_path(std::string(ASCENT_T_DATA_DIR),
+                                       "curv3d_blueprint.cycle_000048.root");
 
-    EXPECT_TRUE(conduit::blueprint::mesh::verify(data,verify_info));
+    // test files:
+    //  - curv3d_blueprint.cycle_000048.root
+    //  - multi_curv3d_blueprint.cycle_000048.root
+    //  - tire_blueprint.cycle_000000.root
+    //  - curv2d_blueprint.cycle_000048.root - we are not sure what will happen for 2D inputs.
```

Comment:
This comment was mainly for you so you know what is available. Now that you have added tests and skeleton tests for these we don't need a note about the options here. There will be more test files coming soon too.